### PR TITLE
add pascalVOC segmentation to dataloader

### PR DESCRIPTION
**Pull request by Louis-Dupont**

### CODE REVIEW

File: src/super_gradients/recipes/dataset_params/pascal_voc_segmentation.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+img_size: 608
```
Comment (shaydeci) [inline src/super_gradients/recipes/dataset_params/pascal_voc_segmentation.yaml:1]: Please rename the file to pascal_voc_segmentation_dataset_params.yaml.

**Review by shaydeci — CHANGES_REQUESTED** [review]
Rename file, resolve conflicts and we can merge.

File: src/super_gradients/recipes/dataset_params/pascal_voc_segmentation.yaml (hunk 1 above)
Comment (Louis-Dupont) [reply src/super_gradients/recipes/dataset_params/pascal_voc_segmentation.yaml]: done

**Review by shaydeci — APPROVED** [review]
LGTM

**Pull request merged**